### DISCUSSION TITLE
Supporting of Quectel UC20 ?

### DISCUSSION THREAD

**Discussion by CpServiceSpb** [IchthysMaranatha/asterisk-chan-quectel#54] [General]
Hi.
Does the Asterisk-Chan-Quectel support Quectel UC20 at the time ?
It is 2g?3g, without 4G.

**Comment by IchthysMaranatha** [comment]
Please read [this](https://github.com/IchthysMaranatha/asterisk-chan-quectel/discussions/15) and see if at^dsci=1 works for your modem

https://github.com/IchthysMaranatha/asterisk-chan-quectel/discussions/15

**Comment by IchthysMaranatha** [comment]
Don't think at^dsci is supported on UC20, there seems to be other commands like at+colp and at+clip for call related urc. Things won't work for you out of the box, you'll need to modify the code to get it to work for your modem
